HW02/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_02_t_02.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+lines = []
+
+File.open('log.txt') do |review_file|
+  lines = review_file.readlines
+end
+
+re = []
+lines.each do |i|
+  ip = i[/\d+\.\d+\.\d+\.\d+/]
+  date = i[%r{\[\d+/\w+/\d{4}\:\d{2}\:\d{2}\:\d{2}\s\+\d+\]}]
+  adr = i[/T .*H/]
```

Comment:
uncommunicative name, please, use the whole name
```suggestion
  address = i[/T .*H/]
```